templates/team-listing.html

```diff
@@ -1,18 +1,46 @@
 {% set unprivileged = not(participant == user.participant or user.ADMIN) %}
 {% set approved_open_teams = participant.get_teams(unprivileged, unprivileged) %}
-
-{% if (user.ADMIN or user.participant == participant) and approved_open_teams %}
+{% if (user.ADMIN or user.participant == participant) or approved_open_teams %}
     <h2>{{ _("Projects") }}</h2>
-    <ul class="team memberships">
-        {% for team in participant.get_teams() %}
-        <li><a href="/{{ team.slug }}/">{{ team.name }}</a></li>
-        {% endfor %}
-    </ul>
-{% elif approved_open_teams %}
-    <h2>{{ _("Projects") }}</h2>
-    <ul class="team memberships">
-        {% for team in approved_open_teams %}
-        <li><a href="/{{ team.slug }}/">{{ team.name }}</a></li>
-        {% endfor %}
-    </ul>
+    {% if user.participant == participant %}
+        <form action="/new" class="apply">
+            <button type="submit">{{ _("Fund Your Project") }}</button>
+        </form>
+    {% endif %}
```

Comment:
Not a bad idea to include this cta here, but if so we should show it in the case where there are zero projects as well.